footprints/batch/tests/test_views.py

```diff
@@ -278,9 +279,8 @@ def test_post(self):
         self.assertEqual(self.record2.footprint, fp2)
 
         self.assertTrue(fp1.book_copy, fp2.book_copy)
-
-        self.assertTrue('2 footprints created' in
-                        response.cookies['messages'].value)
+        messages = [m.message for m in get_messages(response.wsgi_request)]
```

Comment:
Brilliant. I was searching for how the `messages` changed in another application upgrade.